meinberlin/apps/notifications/migrations/0002_add_notification_settings_per_user.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 4.2.16 on 2025-02-03 08:44
+
+from django.db import migrations
+
+
+def create_notification_settings_per_user(apps, schema_editor):
+    NotificationSettings = apps.get_model(
+        "meinberlin_notifications", "NotificationSettings"
+    )
+    User = apps.get_model("meinberlin_users", "User")
+
+    for user in User.objects.all():
+        NotificationSettings.objects.create(
+            user=user,
+            email_newsletter=user.get_newsletters,
+            notify_followers_phase_started=user.get_notifications,
+            notify_followers_phase_over_soon=user.get_notifications,
+            notify_followers_event_upcoming=user.get_notifications,
+            notify_creator=user.get_notifications,
+            notify_creator_on_moderator_feedback=user.get_notifications,
+            notify_initiators_project_created=user.get_notifications,
+            notify_moderator=user.get_notifications,
+        )
```

Comment:
might worth to put this for loop in a try / except block with a warning for when it runs in prod environment